emgapi/migrations/0001_squashed_0043_auto_20220811_1308.py

```diff
@@ -0,0 +1,2489 @@
+# Generated by Django 3.2.12 on 2022-09-30 09:47
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import emgapi.validators
+import uuid
+
+from emgapianns import models as m_models
+
+
+"""
+This squashed migration creates the EMG database as it was at migration 0043 (2022-08-11), 
+whilst populating certain tables to the state they were at that time.
+Those tables are things like Download Description Labels, whose data is (should be) updated through
+migrations.RunPython steps because some code (e.g. import scripts) relies on hardcoded matches to these data.
+
+Some steps are RunSQL and assume SQL dialect that only works on MySQL. 
+These steps have been changed to a helper class (`RunSqlIfMySQL`) so that they can be run on other engines, e.g. SQLite.
+This is relevant to the creation of Full Text Indexes. 
```

Comment:
sqlite full text plugin -> https://www.sqlite.org/fts5.html
note saying we should use it... sqlite has it all hehe

Reply:
@mberacochea Yeh for sure :) My thinking here was that for sqlite test purposes we don't really need the FTIs (tiny data volume) and that for production deployments we would probably need engine-specific migrations for mysql/postgres etc...